src/lib/utils/liquidityMining/index.ts

```diff
@@ -0,0 +1,68 @@
+import { differenceInWeeks } from 'date-fns';
+import { groupBy, mapValues, mergeWith, add } from 'lodash';
+import { bnum } from '..';
+
+import LiquidityMiningV2 from './LiquidityMiningV2.json';
+
+type LiquidityMiningSlot = Record<string, string>;
+
+type LiquidityMiningTierLevel = '1' | '2' | '3';
+
+type LiquidityMiningTier = {
+  BAL: number;
+  slots: LiquidityMiningSlot;
+};
+
+type LiquidityMiningWeek = {
+  start_timestamp: number;
+  end_timestamp: number;
+  tiers: Record<LiquidityMiningTierLevel, LiquidityMiningTier>;
+};
+
+type LiquidityMiningRewards = Record<string, number>;
+
+// Liquidity mining started on June 1, 2020
+function getCurrentLiquidityMiningWeek() {
+  return differenceInWeeks(new Date(), new Date(2020, 5, 1)) + 1;
```

Comment:
@markusbkoch It seems I need to always add `1` to the diff here right? because `differenceInWeeks(new Date(), new Date(2020, 5, 1))` currently gives me `50`.

Reply:
Correct